GANDLF/logging_config.yaml

```diff
@@ -0,0 +1,39 @@
+version: 1
+formatters:
+  detailed:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+  detailed1:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(pathname)s:%(lineno)d - %(message)s"
+filters:
+  warnings_filter:
+    (): logging.Filter
+    name: "py.warnings"
+  info_only_filter:
+    (): GANDLF.utils.gandlf_logger.InfoOnlyFilter
+handlers:
+  stdoutHandler: # only display info level
+    class: logging.StreamHandler
+    level: INFO
+    formatter: detailed1
+    filters: [info_only_filter]
+    stream: ext://sys.stdout
+  stderrHandler: # display warning and above messages
+    class: logging.StreamHandler
+    level: WARNING
+    formatter: detailed1
+    stream: ext://sys.stderr
+  rotatingFileHandler:
+    class: logging.handlers.RotatingFileHandler
+    level: DEBUG
+    formatter: detailed1
+    filename: tmp/gandlf/gandlf.log
+    maxBytes: 10485760
```

Comment:
@VukW What do you think is the best maxBytes for our log file?

Reply:
I believe 1mb you've chosen is ok